src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralService.kt

```diff
@@ -141,16 +142,33 @@ class ReferralService(
 
     /*
      * This is a temporary solution until a robust asynchronous link is created between Interventions
-     * and Delius. Once the asynchronous link is implemented, this feature can be turned off, and instead
-     * Delius will be notified via the normal asynchronous route
+     * and Delius/ARN. Once the asynchronous link is implemented, this feature can be turned off, and instead
+     * Delius/ARN will be notified via the normal asynchronous route
      */
+    submitAdditionalRiskInformation(referral, user)
```

Comment:
there is a big issue here is the subsequent call to community API fails... needs some thought.

Reply:
Consider switching the calls around. Our API doesn't have a legacy service to write to so it's more likely that the Community API call will fail (famous last words I know!)

Reply:
done - this should be sufficient for right now.